operator/docs/forwarding_logs_without_gateway.md

```diff
@@ -0,0 +1,124 @@
+
+* Deploy the Loki Operator and a `lokistack` instance for [OpenShift](./hack_loki_operator.md#hacking-on-loki-operator-on-openshift).
+
+* Deploy the OpenShift Logging Operator from the Operator Hub or using the following command locally:
+
+    ```console
+    make deploy-image deploy-catalog install
+    ```
+  
+* Create a Cluster Logging instance in the `openshift-logging` namespace with only `collection` defined.
+
+    For fluentd:
+
+    ```yaml
+    apiVersion: "logging.openshift.io/v1"
+    kind: "ClusterLogging"
+    metadata:
+      name: "instance"
+      namespace: openshift-logging
+    spec:
+      collection:
+        logs:
+          type: "fluentd"
+          fluentd: {}
+    ```
+
+    For vector:
+
+    ```yaml
+    apiVersion: "logging.openshift.io/v1"
+    kind: "ClusterLogging"
+    metadata:
+      name: "instance"
+      namespace: openshift-logging
+    spec:
+      collection:
+        logs:
+          type: "vector"
+          fluentd: {}
+    ```
+
+* By default, the TLS communication is on for communication with the distributor. Therefore, you need to create a secret. The secret must exist in the openshift-logging namespace, and must have a key `ca-bundle.crt`.
+
+    This key already exists when LokiStack instance got created but as a Config Map. Hence, you need to fetch them and put into a secret file.
+
+    Fetch the `ca-bundle.crt` using:
+
+    ```console
+    kubectl -n openshift-logging get cm lokistack-gateway-lokistack-dev-ca-bundle -o jsonpath="{.data.service-ca\.crt}" | base64 -w 0
+    ```
+  
+* Once secret is fetched, create a new secret file:
+
+    ```yaml
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: loki-distributor-metrics
+      namespace: openshift-logging
+    type: Opaque
+    data:
+      ca-bundle.crt: <CA_BUNDLE>
+    ```
+    
+    where `<CA_BUNDLE>` is the copied value from previous step.
+
+* Create the above secret file in the cluster:
+
+    ```console
+    kubectl create -f <SECRET_FILE_NAME>
+    ```
+
+* Now create a ClusterLogForwarder CR to forward logs to LokiStack:
+
+  ```yaml
+  apiVersion: "logging.openshift.io/v1"
+  kind: "ClusterLogForwarder"
+  metadata:
+    name: "instance"
+    namespace: "openshift-logging"
+  spec:
+    outputs:
+     - name: loki-operator
+       type: "loki"
+       url: https://loki-distributor-http-lokistack-dev.openshift-logging.svc:3100
+       secret:
+         name: loki-distributor-metrics
+       loki:
+         tenantKey: log_type
+    pipelines:
+     - name: send-logs
+       inputRefs:
+       - application
+       - audit
+       - infrastructure
+       outputRefs:
+       - loki-operator
+  ```
+
+  _Note:_ The `tenantKey: log_type` gets resolved as `application`, `audit` or `infrastructure` by fluentd and vector based on the type of logs being collected. This is later used as different tenants for storing logs in Loki.
```

Comment:
We should probably mention that while these tenants look identical to the ones that the gateway uses in the OpenShift configuration but do not map to the tenants used by the gateway, because it uses UUIDs internally.

Reply:
This won't be required once #5624 is merged.

Reply:
That was great timing :slightly_smiling_face: